skipruntime-ts/core/src/api.ts

```diff
@@ -89,9 +89,11 @@ export interface Reducer<V extends Json, A extends Json> {
 export interface Values<T> extends Iterable<T & DepSafe> {
   /**
    * Return the first value, if there is exactly one.
+   * @param options - Optional named parameters
+   * @param options.default - (optional) A default value to return if this is empty
```

Comment:
A `NonEmptyIterator` cannot be empty. `getUnique` will actually only fail if there are more than one value. Isn't it?